redbot/cogs/mutes/mutes.py

```diff
@@ -0,0 +1,1212 @@
+                    with contextlib.suppress(discord.Forbidden):
+                        await query.clear_reactions()
+                await self.config.guild(ctx.guild).sent_instructions.set(True)
+                return True
+
+    @commands.command()
+    @commands.guild_only()
+    @checks.mod_or_permissions(manage_roles=True)
+    async def activemutes(self, ctx: commands.Context):
+        """
+        Displays active mutes on this server.
+        """
+
+        msg = ""
+        to_del = []
+        if ctx.guild.id in self._server_mutes:
+            mutes_data = self._server_mutes[ctx.guild.id]
+            to_rem = []
+            if mutes_data:
+
+                msg += _("__Server Mutes__\n")
+                for user_id, mutes in mutes_data.items():
+                    if not mutes:
+                        to_rem.append(user_id)
+                        continue
+                    user = ctx.guild.get_member(user_id)
+                    if not user:
+                        user_str = f"<@!{user_id}>"
+                    else:
+                        user_str = user.mention
+                    if mutes["until"]:
+                        time_left = timedelta(
+                            seconds=mutes["until"] - datetime.now(timezone.utc).timestamp()
+                        )
+                        time_str = humanize_timedelta(timedelta=time_left)
+                    else:
+                        time_str = ""
+                    msg += _("{member}").format(member=user_str)
+                    if time_str:
+                        msg += _("__Remaining__: {time_left}\n").format(time_left=time_str)
+                    else:
+                        msg += "\n"
+            for _id in to_rem:
+                try:
+                    del self._server_mutes[ctx.guild.id][_id]
+                except KeyError:
+                    pass
+        for channel_id, mutes_data in self._channel_mutes.items():
+            if not mutes_data:
+                to_del.append(channel_id)
+                continue
+            if channel_id in [c.id for c in ctx.guild.channels]:
+                msg += _("__<#{channel_id}> Mutes__\n").format(channel_id=channel_id)
+                for user_id, mutes in mutes_data.items():
+                    if not mutes:
+                        continue
+                    user = ctx.guild.get_member(user_id)
+                    if not user:
+                        user_str = f"<@!{user_id}>"
+                    else:
+                        user_str = user.mention
+                    if mutes["until"]:
+                        time_left = timedelta(
+                            seconds=mutes["until"] - datetime.now(timezone.utc).timestamp()
+                        )
+                        time_str = humanize_timedelta(timedelta=time_left)
+                    else:
+                        time_str = ""
+                    msg += _("{member} ").format(member=user_str)
```

Comment:
There's nothing to translate here.
```suggestion
                    msg += f"{user_str} "
```